flake-info/src/commands/packages-config.nix

```diff
@@ -1,49 +1,4 @@
-{
-  # Ensures no aliases are in the results.
-  allowAliases = false;
-
+import <nixpkgs/pkgs/top-level/packages-config.nix> // {
   # Do *NOT* list unfree packages
   allowUnfree = false;
```

Comment:
Isn't this the default? If we can remove this we can use the file from nixpkgs directly

Reply:
Right, I was a bit confused because <https://github.com/NixOS/nixos-search/pull/343> changed the value, so I thought it might be necessary.

Keeping the file that reimports from nixpkgs could still be valueable to add custom changes to the upstream one? In any case I'll update the commit message and contents later.